azure-resources/DesktopVirtualization/hostPools/recommendations.yaml

```diff
@@ -1,102 +1,79 @@
-﻿- description: (Personal) Create a validation pool for testing of planned updates
-  aprlGuid: 97d4d8c1-eeb4-4506-b338-79a4949c993b
+﻿- description: Create a validation host pool for testing of planned updates
```

Comment:
Confirmed with @danycontre that he intends these to be different recommendations which is why they have new guid